src/pages/Profile/UserSettings/UserSettings.js

```diff
@@ -66,10 +116,46 @@ class UserSettings extends Component {
     this.saveUserSettings(formData)
   }
 
+  areBasemapNamesUnique = (basemaps) => {
```

Comment:
Lodash has a `_uniq` function, so this this whole thing could just be: `return _uniq(map(basemaps, 'name')).length === basemaps.length`